jobs/README.rst

```diff
@@ -1,35 +1,45 @@
-This folder contains several worker jobs.
+This folder contains several scripts to aggregate build information from different sources and keeping it up to date.
 
 .. notes::
 
-    The ``user:pass`` is the Basic auth on Kinto.
+    The ``user:pass`` in the command-line examples is the Basic auth for Kinto.
 
 
-Pulse listener
-==============
+Scrape archives
+===============
 
-Listen to Pulse build and publishes records on a ``builds`` collection.
+Scrape releases on https://archives.mozilla.org and publishes records on a ``archives`` collection.
```

Comment:
Doesn't it also scrape nightlies, or are those considered releases too? Can it really be considered scraping if we're parsing JSON files that expose metadata? Can we describe a little bit about what information is available from this source? (`This source provides information about release and nightly versions, including their target platform, locale, channel, and version, as well as the URL where they are available for download and (for en-US builds) the tree and revision they were built from.`)